static/tests/whisper.js

```diff
@@ -400,71 +196,91 @@ describe('Whisper Tests', function () {
         });
 
         it('ensure symkey exists', function () {
-            keyId1 = node1.shh.generateSymmetricKey();
-            assert.lengthOf(keyId1, 64);
-            expect(node1.shh.hasSymmetricKey(keyId1)).to.equal(true);
+            keyID1 = node1.shh.newSymKey();
```

Comment:
By the way, code style in JS is different from Go's. They write abbreviations CamelCase, as I know.